ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,532 @@
+# Table of Contents <!-- omit in toc -->
+- [Ansible and AVD Overview](#ansible-and-avd-overview)
+  - [Day 1 operation i.e. provisioning a network from scratch](#day-1-operation-ie-provisioning-a-network-from-scratch)
+  - [What is Ansible](#what-is-ansible)
+  - [What is required to run Ansible](#what-is-required-to-run-ansible)
+  - [What is Ansible AVD (Ansible collection for Arista Validated Designs)](#what-is-ansible-avd-ansible-collection-for-arista-validated-designs)
+  - [When to use Ansible AVD and when not to](#when-to-use-ansible-avd-and-when-not-to)
+  - [Change your mindset and culture](#change-your-mindset-and-culture)
+  - [How does Ansible work](#how-does-ansible-work)
+    - [Inventories](#inventories)
+    - [Organizing host and group variables](#organizing-host-and-group-variables)
+    - [Group Variables](#group-variables)
+    - [Playbooks](#playbooks)
+  - [Source of Truth](#source-of-truth)
+  - [How do I use Ansible AVD?](#how-do-i-use-ansible-avd)
+  - [Day 2 Operations](#day-2-operations)
+    - [Day 2 Operations Example](#day-2-operations-example)
+  - [References](#references)
+
+# Ansible and AVD Overview
```

Comment:
Required blank line after headings, https://github.com/DavidAnson/markdownlint/blob/v0.24.0/doc/Rules.md#md022


Same for all below headings